src/components/evaluation/EvaluationTable.vue

```diff
@@ -290,6 +290,9 @@
       </template>
     </v-data-table>
   </div>
+  <div v-else class="no-elegible-sections">
+    No elegible sections
```

Comment:
@cesarvh Just a minor spelling error: should be 'eligible' here and 2 other instances.

Reply:
🤡 what is english. Fixed!